task/buildah/0.1/buildah.yaml

```diff
@@ -257,7 +257,7 @@ spec:
     workingDir: $(workspaces.source.path)
 
   - name: sbom-syft-generate
-    image: quay.io/redhat-appstudio/syft:v0.105.0@sha256:32a9d2007f2b042ceec4ef32fa1d90b8d28141822e7d9748f240da9d55c56601
+    image: registry.redhat.io/rh-syft-tech-preview/syft-rhel9:0.105.0@sha256:8d34c03188cf294a77339b2a733b1f6811263a369b309e6b170d9b489abc0334
```

Comment:
Ah, I will first need to fix the tasks to use Syft the same way as buildah-rhtap

Otherwise, they fail on missing `find`
```
/tekton/scripts/script-1-cbxrs: line 4: find: command not found
```

Reply:
I gave up, buildah-remote makes that too difficult. Just moved the `find` call to a different step.